dataprep/data_connector/schema.json

```diff
@@ -61,6 +61,15 @@
                 "params": {
                     "$ref": "#/definitions/fields"
                 },
+                "pagination": {
+                    "$id": "#/properties/request/properties/pagination",
```

Comment:
Please format this json file.
Please add `required` field, and `"additionalProperties": false` to make this robust.